src/Entity/Programme.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\Common\Collections\Collection;
+
+class Programme
+{
+    private int $id;
+
+    public string $name = '';
+
+    public string $description = '';
+
+    private \DateTime $startDate;
+
+    private \DateTime $endDate;
+
+    private ?User $trainer;
+
+    private Room $room;
+
+    private ?Collection $customers;
```

Comment:
a collection object should not be null. it's an empty collection if it contains nothing